src/components/services-view/services-view.html

```diff
@@ -100,6 +100,9 @@
                           filter-on-keyword="$ctrl.keywordFilterValue"
                           apply-filters="$ctrl.applyFilters($event)">
           </catalog-filter>
+          <div class="col-xs-12" ng-if="$ctrl.filterConfig.appliedFilters.length > 0 && $ctrl.filterConfig.resultsCount === 0">
```

Comment:
class="col-xs-12" should not be needed.

Reply:
I added that because without it then the pfEmptyState doesn't span the full width. Any alternatives that you suggest?

Reply:
<img width="903" alt="screen shot 2017-10-18 at 3 43 08 pm" src="https://user-images.githubusercontent.com/20052391/31739179-1b0adf7e-b41b-11e7-829e-d4955ba0edfb.png">

Reply:
Ah.  I'd be inclined to write a rule for this case.  col-xs-12 works to get the appearance you're looking for, but isn't really intended for this application.

Reply:
Could you clarify what you mean by write a rule for this case?

Reply:
I see what you're saying. I'll go back and fix.

Reply:
At https://github.com/openshift/origin-web-catalog/blob/master/src/styles/services-view.less#L13, add:

```
.pf-empty-state {
  margin: 0 (@grid-gutter-width / 2);
  width: 100%;
}
```

and then change your [markup class](https://github.com/amarie401/origin-web-catalog/blob/02c97bc365b6d9ad77ce76c16205328a0300eded/src/components/services-view/services-view.html#L103) from `col-xs-12` to `pf-empty-state`.

You also need to include &lt;pf-empty-state&gt; for [subcategories](https://github.com/amarie401/origin-web-catalog/blob/02c97bc365b6d9ad77ce76c16205328a0300eded/src/components/services-view/services-view.html#L78)?

![screen shot 2017-10-18 at 4 25 13 pm](https://user-images.githubusercontent.com/895728/31741087-2af9331c-b421-11e7-8e2b-0231a1eed507.PNG)